### DISCUSSION TITLE
Could not create client: could not make register request: POST https://interact.sh/register giving up after 6 attempts

### DISCUSSION THREAD

**Discussion by ghost** [projectdiscovery/nuclei#1386] [Q&A] [answered]
ubuntu@cogent-falcon:~$ interactsh-client -v

[FTL] Could not create client: could not make register request: POST https://interact.sh/register giving up after 6 attempts: Post "https://interact.sh/register": dial tcp: lookup interact.sh: no such host

Interactsh-client always fails with above message, tried upgrade, still not working.

**Comment by forgedhallpass** [comment, marked as answer]
I cannot reproduce this with the latest version:

![image](https://user-images.githubusercontent.com/13679401/146960423-d910afd8-4d5d-4c82-bd72-e5bcbd73e0ab.png)
